backend-implicaction/src/test/java/com/dynonuggets/refonteimplicaction/controller/JobsPostingControllerTest.java

```diff
@@ -126,4 +131,63 @@ void getJobByIdWithoutJwtShouldBeForbidden() throws Exception {
 
         verify(jobPostingService, times(0)).getJobById(anyLong());
     }
+
+    @Test
+    @WithMockUser
+    void archiveJobShouldChangeStatus() throws Exception {
+        JobPostingDto jobPostingDtoAvailable = JobPostingDto.builder()
+                .id(1L)
+                .createdAt(Instant.now())
+                .description("description")
+                .location("Paris")
+                .keywords("toto,yoyo")
+                .salary("1111$")
+                .status(statusAvailable)
+                .build();
+
+        JobPostingDto jobPostingDtoArchived = JobPostingDto.builder()
+                .id(1L)
+                .createdAt(Instant.now())
+                .description("description")
+                .location("Paris")
+                .keywords("toto,yoyo")
+                .salary("1111$")
+                .status(statusArchived)
+                .build();
+
+        given(jobPostingService.archiveOrUnarchiveJobPosting(jobPostingDtoAvailable.getId())).willReturn(jobPostingDtoArchived);
```

Comment:
```suggestion
        given(jobPostingService.archiveOrUnarchiveJobPosting(anyLong())).willReturn(expectedDto);
```
En fait on s'en fiche de la valeur de l'id en paramètre. On veut juste que  ça nous retourne quelque chose donc c'est le `willReturn()` qui compte